src/nebius/aio/channel.py

```diff
@@ -264,12 +275,16 @@ def __init__(
 
         self._free_channels = dict[str, list[GRPCChannel]]()
         self._methods = dict[str, str]()
+        self.user_agent = "nebius-python-sdk/" + version
+        self.user_agent += f" (python/{sys.version_info.major}.{sys.version_info.minor}"
```

Comment:
```suggestion
        self.user_agent += f" (python={sys.version_info.major}.{sys.version_info.minor}"
```

Reply:
In user-agents version is delimitered with a `/`.